.github/workflows/issues_to_jira.yaml

```diff
@@ -0,0 +1,36 @@
+# this workflow requires to provide JIRA webhook URL via JIRA_URL GitHub Secret
+# read more: https://support.atlassian.com/cloud-automation/docs/jira-automation-triggers/#Automationtriggers-Incomingwebhook
+# original code source: https://github.com/beliaev-maksim/github-to-jira-automation
+
+name: Issues to JIRA
+
+on:
+  issues:
+    # available via github.event.action
+    types: [opened, reopened, closed]
+
+jobs:
+  update:
+    name: Update Issue
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create JIRA ticket
+        run: |
+          # Determine Bug/Enhancement
+          if ${{ contains(github.event.*.labels.*.name, 'Bug') }}; then
+            type=bug
+          else
+            type=story
+          fi
+
+          # send JIRA request
+          data=$( jq -n \
+                  --arg title "${{github.event.issue.title }}" \
+                  --arg url "${{ github.event.issue.html_url }}" \
+                  --arg submitter "${{ github.event.issue.user.login }}" \
```

Comment:
Is the submitter the reporter field? If so, what happens when the user doesn’t match a user available on JIRA? Does it put an empty submitter?

Reply:
it is just a link to a user. No actual user assigned

Reply:
Thanks for the explanation!